core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -348,10 +355,12 @@ abstract class SbtPrecogBase extends AutoPlugin {
       }.toString,
       trickleGitConfig := {
         import sbttrickle.git.GitConfig
-        (sys.env.get("GITHUB_ACTOR"), sys.env.get("GITHUB_TOKEN")) match {
+        val baseConf = (sys.env.get("GITHUB_ACTOR"), sys.env.get("GITHUB_TOKEN")) match {
```

Comment:
Do we still need `GITHUB_ACTOR`?

Reply:
I guess we could use underscore instead, but the API on trickle is not
github-specific, so it still requires a user. Github workflow is still
setting it.